ivy/functional/backends/jax/linear_algebra.py

```diff
@@ -384,7 +384,6 @@ def tensordot(
     axes: Union[int, Tuple[Sequence[int], Sequence[int]]] = 2,
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
-    x1, x2 = promote_types_of_inputs(x1, x2)
```

Comment:
We shouldn't be removing this - it's needed for when `x1` and `x2` have different dtypes. It will use ivy's type promotion table to do the update so that we have a consistent return dtype for all the backends,